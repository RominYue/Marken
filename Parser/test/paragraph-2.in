*   [Miscellaneous](#misc)
    *   [Backslash Escapes](#backslash)
    *   [Automatic Links](#autolink)


**Note:** This document is itself written using Markdown; you
can [see the source for it by adding '.text' to the URL][src].

  [src]: /projects/markdown/syntax.text

* * *