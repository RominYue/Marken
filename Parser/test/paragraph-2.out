<ul><li><a href="#misc">Miscellaneous</a>
<ul><li><a href="#backslash">Backslash Escapes</a></li>
<li><a href="#autolink">Automatic Links</a>

</li></ul></li></ul>
<p><strong>Note:</strong> This document is itself written using Markdown; you
can <a href="/projects/markdown/syntax.text">see the source for it by adding '.text' to the URL</a>.</p>



<hr>
